lib/IgnorePlugin.js

```diff
@@ -34,12 +45,12 @@ class IgnorePlugin {
 		return this.contextRegExp.test(context);
 	}
 
-	/*
-	 * Returns true if result should be ignored.
-	 * false if it shouldn't.
-	 *
-	 * Not that if "contextRegExp" is given, both the "resourceRegExp"
+	/**
+	 * Note that if "contextRegExp" is given, both the "resourceRegExp"
 	 * and "contextRegExp" have to match.
+	 *
+	 * @param {TODO} result result
```

Comment:
Not sure what exactly to put here, would appreciate some guidance

Reply:
I see it depends on #7299

Reply:
After digging a bit deeper into it turns out this depends on `tapable` typing, or am I missing something?

Are we planning to add types in `tapable` as well or shall we leave this as `TODO` for the time being?

/cc @sokra @ooflorent @mohsen1